### PR TITLE
Introduce a `services` directory for non-critical services

### PR DESCRIPTION
**Pull request by petr-muller**
The `services` directory is intended to be a middle-ground between `core-services` (where important, universally useful services live) and `projects` (wild west of legacy and stuff that may or not work).

A separate set of jobs will handle this config but otherwise the principles and quality criteria are the same as for `core-services`.

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+	applyconfig --config-dir services
+
 core:
 	applyconfig --config-dir core-services --confirm=true
 
+services:
+	applyconfig --config-dir services --confirm=true
+
+all-admin: core-admin services-admin
+all: core-admin core services-admin services
```
Comment (bbguimaraes) [inline Makefile]: These can be reversed and moved above line 3 so `make` == `make all`.

Comment (petr-muller) [reply Makefile]: Done.

**Comment by stevekuznetsov** [comment]
Do you want the new jobs in this PR, too?

**Pull request merged**